### DISCUSSION TITLE
How to remove the adsense, followit and subscribe_mail in widgets

### DISCUSSION THREAD

**Discussion by NanoWraith** [ppoffice/hexo-theme-icarus#1151] [Q&A]
Errors in detail are as follows

![image](https://user-images.githubusercontent.com/118002757/202859709-a4d385eb-8aab-402b-9be4-9b7778432689.png)

![image](https://user-images.githubusercontent.com/118002757/202859881-83cf0056-6038-404a-9f75-2f376fb1080b.png)

**Comment by NanoWraith** [comment]
btw if I comment or delete three types of this, I can't visit the website, what should I do?

**Reply by ppoffice** [reply]
Probably you forgot to comment the `-` in the `subscribe_email` block.

**Reply by NanoWraith** [reply]
oh thx, hope you have a nice weekend :-)
